src/MCRun.jl

```diff
@@ -1,17 +1,18 @@
 module MCRun
 
 export MCState
-export metropolis_condition, mc_step!, mc_cycle!, ptmc_run!
-export atom_move!
+export metropolis_condition, mc_step!, mc_cycle!,ptmc_cycle!, ptmc_run!,save_states,save_params,save_results
+export atom_move!,update_max_stepsize!
```

Comment:
exporting more functions for redefinition in parallelrun

Reply:
?